packages/lib/src/ui/styles/theme.ts

```diff
@@ -17,39 +17,21 @@ export const theme = {
     darkest: "var(--darkest)",
   },
   size: {
-    small: "var(--small-size)",
-    base: "var(--base-size)",
+    small: 1,
+    base: 1.2,
   },
   fonts: {
     text: "'Open Sans', verdana, sans-serif",
     accent: "themix, verdana, sans-serif",
     headings: "skautbold, verdana, sans-serif",
   },
   timing: {
-    fast: "0.2s ease-in-out",
-    slow: "0.35s ease-in-out",
-  },
-  navSize: {
-    horizontal: {
-      height: "5rem",
-      linkSize: "1.5rem",
-      subLinkSize: "1.4rem",
-      linkPadding: "0.55em",
-      padding: "0.3em",
-      subLinkPadding: "0.4em",
-      subLinkHPadding: "0.5em",
-      subLinkImageSize: "4rem",
-      subLinksMaxWidth: "300px",
-      linkSidePadding: "2rem",
-    },
-    vertical: {
-      height: "4rem",
-      linkHeight: "3rem",
-      buttonSize: "1.5rem",
-    },
+    fast: ".2s ease-in-out",
+    slow: ".35s ease-in-out",
   },
 }
+
 export const min = (breakpoint: "s" | "m" | "l") =>
-  `(min-width: ${theme.breakpoints[breakpoint]}px)`
+  `(min-width: ${theme.breakpoints[breakpoint]}rem)`
 export const max = (breakpoint: "s" | "m" | "l") =>
-  `(max-width: ${theme.breakpoints[breakpoint] - 1}px)`
+  `(max-width: ${theme.breakpoints[breakpoint]}rem)`
```

Comment:
Když bude teď šířka viewportu rovna hodnotě nějakého breakpointu, tak se budou aplikovat styly `min` i `max`, což neudělá dobrotu. Proto tam bylo to `- 1px`.